source/catalog.html

```diff
@@ -0,0 +1,157 @@
+<!DOCTYPE html>
+<html class="catalog" lang="ru">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Каталог</title>
+  <link rel="icon" href="./favicon.ico">
+</head>
+
+<body class="catalog__body">
+
+
+  <!-- Хедер -->
+  <header class="main-header">
+    <nav class="main-nav">
+      <a class="main-nav__logo" href="#">
+        <img src="./img/logo.svg" alt="Логотип Седоны." width="185" height="96">
+      </a>
+      <button class="main-nav__toogle" type="button">
+        <span class="visually-hidden">Открыть меню</span>
+      </button>
+      <div class="main-nav__wrapper">
+        <ul class="main-nav__list">
+          <li class="main-nav__list__item main-nav__list__item--current">
+            <a>Главная</a>
+          </li>
+          <li class="main-nav__list__item">
+            <a href="catalog.html">Фото и видео</a>
+          </li>
+          <li class="main-nav__list__item">
+            <a href="form.html">Форма Отзыва</a>
+          </li>
+          <li class="main-nav__list__item">
+            <a class="#">HTML ACADEMY</a>
+          </li>
```

Comment:
Тут не правильно используется БЕМ. Нижний подчеркивания используются один раз
```suggestion
          <li class="main-nav__list-item main-nav__list-item--current">
            <a>Главная</a>
          </li>
          <li class="main-nav__list-item">
            <a href="catalog.html">Фото и видео</a>
          </li>
          <li class="main-nav__list-item">
            <a href="form.html">Форма Отзыва</a>
          </li>
          <li class="main-nav__list-item">
            <a class="#">HTML ACADEMY</a>
          </li>
```

Reply:
Done